core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -59,177 +59,195 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
     requireUserInfo(Option(otelContext)) { userInfo =>
       val ctx = RawlsRequestContext(userInfo, Option(otelContext))
       pathPrefix("billing" / "v2") {
-
-        pathPrefix(Segment) { projectId =>
-          pathEnd {
+        pathPrefix("spendReport") {
```

Comment:
This file must've been hit by `scalafmt` for some reason; adding in this new `spendReport` route is all I did